node_modules/gh-orgunit/lib/internal/import.js

```diff
@@ -0,0 +1,594 @@
+/**
+ * Copyright (c) 2015 "Fronteer LTD"
+ * Grasshopper Event Engine
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+var _ = require('lodash');
+
+var AppsConstants = require('gh-apps/lib/constants');
+var GrasshopperUtil = require('gh-core/lib/util');
```

Comment:
I've seen a mix of `GrassHopperUtil` and `GrasshopperUtil` nomenclature. I agree with this one, so lets try and standardize on this moving forward and eventually submit a sweeping PR